src/modules/devops-guide/pages/system-requirements.adoc

```diff
@@ -124,11 +124,19 @@ If you enable OAuth2 or OIDC sign-up or access external services via automation
 
 == Domain and Hostname
 
-.The most common setup involves two domains -- API and web app:
-* Hostname for Web application,
-* hostname for API server.
+The most common way to set up your {PRODUCT_NAME}  instance is by hosting the web application and the API server on one domain and providing;
 
-If you already own a domain, you can add two or more hosts and use them to access your {PRODUCT_NAME} instance.
+* A hostname for the Web application,
+
+* A hostname for the API server.
+
+Alternatively, if needed, the setup could be split into two and host the web application and the API server separately. 
+This would require you to have two domains.
+
+[NOTE]
+====
+If you already own a domain, you can add two or more hosts and use them to access your {PRODUCT_NAME}  instance.
```

Comment:
Also, rewrite this line to this:

If you decide for a split setup, Corteza does not require different domains for it services. You can use different hosts or subdomains and point them to each service, like this web.example.tld and api.example.tld